### ISSUE TITLE
README.md should be written in plain english - some reformatting required

### ISSUE THREAD

**Issue by mcm1957** [iobroker-community-adapters/ioBroker.worx#703]
Please consider to translate all german parts of README.md to english.

README:md shouöld be written in planin english as it is translated automatically at iobroker website. This translation does nogt work if README.md comntains mu,ltiple languages.

Description using other languages should be located at doc/en, doc/de, ..

In addtion some reformatting is required as lists are not displayed correctly

**Comment by Lucky-ESA** [comment]
@mcm1957 
Done with the next update (Pull Request)

Gruß//Lucky

**Comment by Lucky-ESA** [comment]
@mcm1957 
Can you check the new readme again?

**Comment by mcm1957** [comment]
OK
